lib/core/src/lib.rs

```diff
@@ -27,10 +27,18 @@ pub mod system {
 
 use display::world::World;
 use wasm_bindgen::prelude::*;
+use basegl_system_web::keyboard_engine::KeyboardEngine;
 
 #[wasm_bindgen(start)]
 pub fn start() {
     let world = World::new();
     world.add_workspace("canvas");
     world.start();
+
+    let mut ke = KeyboardEngine::from_tag_name("input", true);
+    ke.capture("a".to_string(), Box::new(|| { web_sys::console::log_1(&"Key pressed: a".into()); }) );
+    ke.capture("Shift-a".to_string(), Box::new(|| { web_sys::console::log_1(&"Key pressed: Shift-a".into()); }) );
+    ke.capture("C-Shift-a".to_string(), Box::new(|| { web_sys::console::log_1(&"Key pressed: Control-Shift-a".into()); }));
+    ke.capture("C-Space".to_string(), Box::new(|| { web_sys::console::log_1(&"Key pressed: Control-Space".into()); }));
+    ke.capture("A-Space".to_string(), Box::new(|| { web_sys::console::log_1(&"Key pressed: Alt-Space".into()); }));
```

Comment:
Btw, I know this could sound hard and not natural, but lets try using the code style that we've got described in our docs. The earlier you start using it, the easier it will be for all of us read the same codebase. For example, a lot of your lines are over 80 chars length. Keeping the limit of 80 chars makes the code easier to read vertically, however, it often requires to think about better var names as well, so using it from the beginning makes sense, as it will impact the style (and naming) of your code.

Reply:
Sure, I thought that you only use `rustfmt`, when I did commit I had seen style guide. I'm going to pass through it and edit the code